### PR TITLE
Better pluralization of error messages for invalid resource bindings, in single/multi-patient tests

### PR DESCRIPTION
**Pull request by okeefm**
The error messages for "invalid bindings" were not well pluralized (they would always use the plural, even if only one binding was invalid), so I used the ActiveSupport `String#pluralize` function to properly pluralize words as appropriate.

Pluralize rubydoc is here: https://www.rubydoc.info/docs/rails/4.1.7/String:pluralize

Screenshot: here's what the new error message looks like in action:

Screen Shot 2020-06-01 at 9 25 56 AM

Pull requests into Inferno require the following items to be completed. Submitter and reviewer 
should check the relevant check boxes when the associated item is done. For items that are not 
applicable, note it's not applicable ("N/A") and check the box. For example, external Pull 
Requests do not require ticket links.

**Submitter:**
- [x] This pull request describes why these changes were made
- [x] Internal ticket for this PR: N/A
- [x] Internal ticket links to this PR N/A
- [x] Internal ticket is properly labeled (Community/Program) N/A
- [x] Internal ticket has a justification for its Community/Program label N/A
- [x] Code diff has been reviewed for extraneous/missing code
- [x] Tests are included and test edge cases
- [x] Tests/code quality metrics have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/modules/uscore_v3.1.0/bodyheight_sequence.rb [hunk 1]
```diff
@@ -541,8 +541,7 @@ def patient_ids
           invalid_bindings.each { |invalid| invalid_binding_resources << "#{invalid[:resource]&.resourceType}/#{invalid[:resource].id}" }
           invalid_binding_messages.concat(invalid_bindings.map { |invalid| invalid_binding_message(invalid, binding_def) })
         end
-        assert invalid_binding_messages.blank?, "#{invalid_binding_messages.count} invalid required binding(s) found in #{invalid_binding_resources.count} resources:" \
-                                                "#{invalid_binding_messages.join('. ')}"
+        assert invalid_binding_messages.blank?, "#{invalid_binding_messages.count} invalid required #{'binding'.pluralize(invalid_binding_messages.count)}" " found in #{invalid_binding_resources.count} #{'resource'.pluralize(invalid_binding_resources.count)}: " "#{invalid_binding_messages.join('. ')}"
```
Comment (Jammjammjamm) [inline lib/modules/uscore_v3.1.0/bodyheight_sequence.rb:544]: It doesn't look like these are being generated in the intended way. Maybe the template needs `\\` instead of just `\`?

**Comment by yunwwang** [comment]
Code looks good. Do you have a screenshot showing the single binding error? I could not find an example to test this.

File: lib/modules/uscore_v3.1.0/bodyheight_sequence.rb (hunk 1 above)
Comment (okeefm) [reply lib/modules/uscore_v3.1.0/bodyheight_sequence.rb:544]: Yeah, that does appear to be what we were using before. Thanks for the catch; I'll re-generate.

**Comment by okeefm** [comment]
I've been trying to generate a single-patient binding error. I may have to go in and manually inject one via byebug or something...getting one into the reference server is proving problematic.
\

**Comment by okeefm** [comment]
Here's an example of the single-patient error (with a dummy message): Screen Shot 2020-06-01 at 12 57 24 PM

**Comment by arscan** [comment]
Looks good.  I checked and there are no issues w/the generated files merging in w/the other recent us core updates.
